apps/dolly-frontend/src/main/js/src/components/bestilling/sammendrag/kriterier/BestillingKriterieMapper.js

```diff
@@ -613,6 +560,47 @@ export function mapBestillingData(bestillingData, bestillingsinformasjon) {
 			data.push(sivilstandData)
 		}
 
+		if (forelderBarnRelasjon) {
```

Comment:
Savner litt mer info på denne 😁  Ihvertfall relatertPersonsRolle og relatertPerson. Kunne kanskje vært satt opp slik som sivilstand, sånn at vi får med et ekspanderbart felt for nyRelatertPerson? 🤓

Reply:
Ja godt poeng, ser den er litt skranten 😅